__tests__/database/schema/ChatSchema.test.ts

```diff
@@ -292,6 +302,14 @@ describe('TESTING ChatSchema', () => {
           text: 'message',
           type: 'type',
           message: 'message',
+          attachments: [
+            {
+              type: 'image',
+              thumb_url: 'thumb_url',
+              asset_url: 'asset_url',
+              myCustomField: 'image'
+            }
+          ],
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This part of the test seems to be checking whether the `attachments` field can handle complex objects with multiple properties. This is good for testing the robustness of your code.



<!-- This is an auto-generated comment by OSS CodeRabbit -->